tests/test_worker.py

```diff
@@ -23,6 +23,12 @@
     Worker,
 )
 
+_default_shared_state_manager = (
+    temporalio.worker.SharedStateManager.create_from_multiprocessing(
+        multiprocessing.Manager()
+    )
+)
```

Comment:
We made this eager at the top of tests because where we originally had it, as a local inside a function, was causing all local vars to be copied during `fork` (called implicitly by `multiprocessing.Manager`) which included the Python client which has a reference to the Rust client and when that Python client was GC'd from the child process, the Rust client was dropped, causing a file descriptor mismatch.

I have opened https://github.com/temporalio/sdk-python/issues/11 to be more explicit about how fork works with this SDK.